src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs

```diff
@@ -65,6 +64,9 @@
                                               [:div {} message])
                                          message-list)])}
                  {:header "Workflow ID" :starting-width 300
+                  :as-text
+                    (fn [workflow] (let [workflowId (workflow "workflowId")] (str workflowId)))
```

Comment:
This can just be `(fn [workflow] (workflow "workflowId"))` or `#(% "workflowId")`